packages/lesswrong/components/common/SectionFooter.tsx

```diff
@@ -27,6 +27,7 @@ const styles = (theme: ThemeType) => ({
   root: {
     display: "flex",
     justifyContent: "flex-end",
+    alignItems: "center",
```

Comment:
It's probably fine but check that this doesn't do anything weird to the vertical alignment on other pages, particulary the footer under post-lists on the front page